solr/CHANGES.txt

```diff
@@ -159,6 +159,8 @@ Other Changes
 * SOLR-15342: SolrJ: Split ZooKeeper based dependencies off to a solr-solrj-zookeeper module /
   dependency. (David Smiley, Haythem Khiri)
 
+* SOLR-16296: Remove the use of XmlConfigFile by QueryElevationComponent. (Haythem Khiri)
```

Comment:
Would it be worth adding here that this work also changes the use of xpath when parsing elevator.xml?

Reply:
"and XPath" -- sure; it's just two more words.